src/style.css

```diff
@@ -0,0 +1,98 @@
+/* html {
```

Comment:
I have problems with deploy 
Every time I run  "npm run deploy" I hangs for too long 
Don't know what I do wrong
<img width="684" alt="Снимок экрана 2023-04-26 в 17 04 52" src="https://user-images.githubusercontent.com/107880739/234619922-092b99a7-5ade-4eba-8536-c566a59a63d7.png">

Reply:
I try to follow the steps from instruction over and over but it doesn't work 
I cannot publish the website, that's why links doesn't work I suppose 
Below you'll find the steps I follow 
I exit command by control+C because it stays like this for a long time.. 
And then I get an error
Could you help me, please, I don't know what I do wrong
<img width="602" alt="Снимок экрана 2023-04-26 в 18 48 00" src="https://user-images.githubusercontent.com/107880739/234646636-9e0ea829-fdeb-4b31-8bd7-021b1560c570.png">